tt/request_params.html.tt

```diff
@@ -60,15 +60,27 @@
     <th>[% capt.job_split_strategy %]</th>
     <td style="padding: 0;">
       <div class="limits [% si.style_classes.job_split_strategy %]">
-      <textarea name="job_split_strategy" rows="2" cols="25" class="bordered">[% si.job_split_strategy | html %]</textarea>
+      <textarea name="job_split_strategy" rows="4" cols="50" class="bordered">[% si.job_split_strategy | html %]</textarea>
       </div>
       <input type="checkbox" name="set_job_split_strategy" value="1"
         [%- 'checked="checked"' IF si.req_overidden_limits.job_split_strategy %] />
     </td>
   </tr>
   <tr>
     <th>[% capt.judge %]</th>
-    <td>[% select('judge', si.judges) %] <input type="checkbox" name="set_judge" value="1" /></td>
+    <td class="nested">
+      <table class="internal_borders">
+        <tr>
+          <td>
+            [% select('judge', si.judges) %]
+            <input type="checkbox" name="set_judge" value="1" />
+          </td>
+          <td>
+            <label><input type="checkbox" name="this_judge_only" value="1" /> [% capt.this_judge_only %] </label>
```

Comment:
This is slightly misleading. This checkbox could be set without assigning a specific judge,
so "single judge" or even "do not split" is more precise?
Also, consider auto-setting this together with set_judge from combobox's onselect event

Reply:
Unneeded space before closing `label`.